plan/common_plans.go

```diff
@@ -195,6 +194,7 @@ func (e *Execute) getPhysicalPlan(ctx sessionctx.Context, is infoschema.InfoSche
 }
 
 func (e *Execute) rebuildRange(p Plan) error {
+	sctx := p.context()
 	sc := p.context().GetSessionVars().StmtCtx
```

Comment:
If sc is not nil, should we use it as sctx?

Reply:
we can't, they implement different interfaces, can not be mix-used.